src/devtools/client/webconsole/components/Output/message-types/EvaluationResult.js

```diff
@@ -17,14 +17,23 @@ EvaluationResult.propTypes = {
   timestampsVisible: PropTypes.bool.isRequired,
   maybeScrollToBottom: PropTypes.func,
   open: PropTypes.bool,
+  topLevelClassName: PropTypes.string,
 };
 
 EvaluationResult.defaultProps = {
   open: false,
 };
 
 export default function EvaluationResult(props) {
-  const { dispatch, message, timestampsVisible, maybeScrollToBottom, open, isPaused } = props;
+  const {
+    dispatch,
```

Comment:
Generally I have this preference to!

I think I'd argue _against_ it in this case for _pre-existing code_ because it causes churn.

I'd love for us to _standardize_ this (with an auto-fix Lint rule) in which case I'd be thrilled to do it.